Food Receipe/Pages/index.html

```diff
@@ -1,13 +1,96 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="../styles.css">
-    <script src="../index.js"></script>
-    <title>Document</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles.css" />
+    <script
+      src="https://kit.fontawesome.com/bb8e411a96.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="../index.js" defer></script>
+    <title>Recipe Finder</title>
+  </head>
+  <body>
+    <header>
+      <h1>Recipe Hub</h1>
+      <p>Search recipes by ingredients</p>
+    </header>
+    <main>
+      <section class="search">
```

Comment:
can you change the naming  to search-section. This looks more clean

Reply:
done. good idea